c++/src/capnp/CMakeLists.txt

```diff
@@ -291,3 +291,8 @@ if(BUILD_TESTING)
     add_test(NAME capnp-evolution-tests-run COMMAND capnp-evolution-tests)
   endif()  # NOT CAPNP_LITE
 endif()  # BUILD_TESTING
+
+if(DEFINED ENV{LIB_FUZZING_ENGINE})
+  add_executable(fuzz_run fuzz-run.c++ test-util.c++ test-util.h ${test_capnp_cpp_files} ${test_capnp_h_files})
```

Comment:
Ugh sorry, just noticed the cmake file is still calling this fuzz_run, can we make it capnp_llvm_fuzzer_testcase here too? And rename the source file to llvm-fuzzer-testcase.

Reply:
Done.
Do you want me to squash the commits ?

Reply:
I was also hoping the source file could be renamed to `llvm-fuzzer-testcase.c++`, but it looks like it's still `fuzz-run.c++` right now.

I can tell GitHub to squash when I merge.

Reply:
Renamed